.github/workflows/unix_python_matrix_test.yml

```diff
@@ -7,11 +7,12 @@ on:
 
 jobs:
   pyomo-mac-tests:
-    name: py${{ matrix.python-version }}
-    runs-on: macos-latest
+    name: ${{ matrix.os }}/py${{ matrix.python-version }}
```

Comment:
Is there a way to shorten the name to be just `osx/3.5` or `linux/3.6`?

Reply:
Yup! See new commit.